### PR TITLE
Fix AdminUrlPrefix in blog recipe

### LINKED ISSUE #7924: AdminUrlPrefix wrong config
In some links in the admin panel aren't working as expecting: 

For example, I set mine to be "olimpo" but some menu links still point to "admin" instead:

image

After think about it, I will create a PR at EOD.

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #7924

### CODE REVIEW

**Comment by agriffard** [comment]
The Blog Tests are failing in CI.

File: src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
   // The variables are evaluated the first time they are accessed, and reused across steps
   "variables": {
+    "adminUrlPrefix": "[js:configuration('OrchardCore_Admin:AdminUrlPrefix']",
```
Comment (deanmarcussen) [inline src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]:
```suggestion
    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin']",
```

**Review by deanmarcussen — COMMENTED** [review]
You need to pass a default value as well @hishamco

**Comment by hishamco** [comment]
Blocked by #7928 I will update this today

File: src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]: I was thinking about that but while the `appsettings.json` has it I thought it is not needed

Comment (deanmarcussen) [reply src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]: I don't know how many times I have to tell you @hishamco the source on github is not what is used in a NuGet package.

Comment (hishamco) [reply src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]: AFAIK the host app should have `appsettings.json` Am I right? regardless of the using the source or NuGet packages

File: src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
   // The variables are evaluated the first time they are accessed, and reused across steps
   "variables": {
+    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin']",
```
Comment (deanmarcussen) [inline src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]:
```suggestion
    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin')]",
```

Comment (deanmarcussen) [reply src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]: Please can you try this locally, and validate that the menu items have picked up both the default, and a custom one.

Nothing in our tests will validate that this has been successful.

Comment (hishamco) [reply src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json:14]: Sure, I just facing an npm issue, I will review this PR today

**Review by deanmarcussen — APPROVED** [review]

**Pull request merged**